Modules/Projects/Core/Script_MiscUtilities.lua

```diff
@@ -44,6 +44,27 @@ function LogGsub(key, vars, color)
     LogRaw(strval, color)
 end
 
+function PreySense(pred, style, does_digest)
+    if(Player.PreySense)then
```

Comment:
- The parentheses aren't needed here.
- Indentation is all over the place in this function. Please ensure you indent everything consistently following an enclosing scope.
- Be sure to also replace the style constants (`style == 0` etc) lines with the new `EVoreStyle` enum values -- that's what they're for.